openedx/core/lib/block_structure/transformers.py

```diff
@@ -1,9 +1,11 @@
 """
 Module for a collection of BlockStructureTransformers.
 """
+from functools import reduce as functools_reduce
```

Comment:
I think I did that originally to address a name-conflict quality issue.  But it looks like it's no longer an issue.